src/main/java/org/opensearch/security/dlic/rest/validation/EndpointValidator.java

```diff
@@ -156,21 +156,19 @@ default ValidationResult<SecurityDynamicConfiguration<?>> validateRoles(
     default ValidationResult<SecurityConfiguration> isAllowedToChangeEntityWithRestAdminPermissions(
```

Comment:
Looks like this is used by the RolesApi & RolesMappingAPI could we expand the test coverage for those scenarios too?